--- a/net/minecraft/advancements/criterion/LocationPredicate.java
+++ b/net/minecraft/advancements/criterion/LocationPredicate.java
@@ -12,6 +12,7 @@
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.registry.Registry;
+import net.minecraft.world.DimensionType;
 import net.minecraft.world.World;
 import net.minecraft.world.biome.Biome;
 import net.minecraft.world.gen.feature.structure.Structure;
